src/services/generate-default-data.ts

```diff
@@ -104,6 +106,9 @@ export const generateDate = ({
         const offset = date.getTimezoneOffset()
         return new Date(date.getTime() + offset * 60000)
       }
+      if (dateElementIdUsed && daysOffset !== undefined) {
```

Comment:
As discuss, we dont need this, we can just add the offset to the date if it is passed in.